src/eicr_anonymization/anonymizer.py

```diff
@@ -192,18 +227,19 @@ def anonymize_II_value(self, element: Element):
             ):
                 replacement += "X"
             elif char.isdigit():
-                replacement += str(randint(0, 9))
+                replacement += str(random.randint(0, 9))
             elif char.isalpha() and char.islower():
-                replacement += choice(ascii_lowercase)
+                replacement += random.choice(ascii_lowercase)
```

Comment:
with the change in random state-ing, you could probably change all these namespaced calls back if you want (I don't care either way)